src/MCPServer/share/mysql_dev_9647_archivesspace_inherit_notes.sql

```diff
@@ -0,0 +1,5 @@
+-- Add column inherit_notes column to administration_archivesspaceconfig
+ALTER TABLE `administration_archivesspaceconfig` ADD COLUMN `inherit_notes` bool NOT NULL AFTER `repository`;
```

Comment:
Ok so if I understood correctly, my migration is going to add the column and we deal with upgrades for existing users later? Out of curiosity, how have you dealt with similar situations before?